test/e2e/beta/from-import-beta-ui.spec.js

```diff
@@ -110,6 +112,12 @@ describe('Using MetaMask with an existing account', function () {
       await delay(largeDelayMs)
     })
 
+    it('clicks the "No thanks" option on the metametrics opt-in screen', async () => {
+      const optOutButton = await findElement(driver, By.css('.btn-default'))
+      optOutButton.click()
+      await delay(largeDelayMs)
+    })
+
     it('imports a seed phrase', async () => {
       const [seedTextArea] = await findElements(driver, By.css('textarea.first-time-flow__textarea'))
       await seedTextArea.sendKeys(testSeedPhrase)
```

Comment:
<img width="894" alt="screen shot 2019-02-28 at 10 50 17 am" src="https://user-images.githubusercontent.com/13376180/53583373-13a7ed80-3b47-11e9-8c7f-70a7f9bcc853.png">

The TOS needs to be clicked after the seed phrase and password is entered.